src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -277,9 +279,11 @@ private void apply(RoutingContext routingContext) {
     });
   }
 
-  private CompletableFuture<Result<JsonObject>> buildJsonResult(String policyId){
-    return CompletableFuture.completedFuture(succeeded(new JsonObject().put(getPolicyIdKey(),
-      policyId)));
+  private CompletableFuture<Result<JsonObject>> buildJsonResult(Pair<String, List<String>> policyIdAndConditionsPair) {
```

Comment:
I think it would be easier to understand this (and related methods) if they used explicit types rather than pairs of string and lists (or similar).

Reply:
what do you mean? to create an entity as wrapper? with fields String id and List<String> conditions?

Reply:
Yes, I mean a class with properties for the policy ID and the rules conditions.

I think the conditions themselves can also be named properties as the 3 that are being used are known and have clear names, rather than using magic keys.

Reply:
Updated.
I have created CirculationRulePolicyIdEntity.